### DISCUSSION TITLE
Which version am I on, and/or what's the release schedule?

### DISCUSSION THREAD

**Discussion by lindhe** [zigpy/zha-device-handlers#3213] [Q&A] [answered]
Hi!

I'm very much looking forward to https://github.com/zigpy/zha-device-handlers/pull/3169 which seems to have been part of the [0.0.116 release](https://github.com/zigpy/zha-device-handlers/releases/tag/0.0.116). But I'm not sure which version I'm on or what the release schedule is like (if there is one).

I'm currently on Home Assistant version 2024.6.3, but I don't know how to determine the version of ZHA or, more importantly in my case, the version of zha-device-handlers I'm on.

Thanks a bunch for all the work on this project, it's incredibly useful to me! :heart:

**Comment by Hedda** [comment, marked as answer]
> I'm currently on Home Assistant version 2024.6.3, but I don't know how to determine the version of ZHA or, more importantly in my case, the version of zha-device-handlers I'm on.

The ZHA integration is part of Home Assistent core so does not have its own version, but you check check version of zha-device-handlers by looking for merged version of ”zha-quirks” (in the core repository of Home Assistent) as that is what the ZHA Device Handlers package is called on PyPI:

https://github.com/home-assistant/core/pulls?q=is%3Apr+is%3Aclosed+zha-quirks+

https://pypi.org/project/zha-quirks/

In your case the [0.0.116 release] (https://github.com/zigpy/zha-device-handlers/releases/tag/0.0.116) made it into milestone 2024.6.0b0 (i.e. 2024.6 beta 0)

* https://github.com/home-assistant/core/pull/118393

* https://github.com/home-assistant/core/milestone/668
